### PR TITLE
update dialect xsd up to date

### PR DESCRIPTION
**Pull request by jsun28**
Update this dialect xsd as latest near, changes are from near file history 6-10(5 revisions in total)

### CODE REVIEW

**Review by rosswbrown — CHANGES_REQUESTED** [review]
Some of these will be in 2021.3 and others 2021.4.

Once master has been updated to 2021.2, we'll create 2021.3 and .4 branches here.

**Comment by jsun28** [comment]
> Some of these will be in 2021.3 and others 2021.4.
> 
> Once master has been updated to 2021.2, we'll create 2021.3 and .4 branches here.

Thanks, I changed the target branch to dev-2021.3, and looks like only revision 10 in near is not in dev-2021.3, so removed that change.

File: validation/tdd_latest.xsd [hunk 1]
```diff
@@ -548,6 +558,7 @@ as passed-in parameters.
       <xs:element maxOccurs="unbounded" minOccurs="0" name="part">
         <xs:complexType>
           <xs:attribute name="name" type="JoinType-ST" use="required"/>
+          <xs:attribute name="value" type="xs:string" />
```
Comment (rosswbrown) [inline validation/tdd_latest.xsd:561]:
```suggestion
          <xs:attribute name="value" type="xs:string" /> <!-- added 2020.3 -->
```

File: validation/tdd_latest.xsd [hunk 2]
```diff
@@ -633,6 +647,7 @@ as passed-in parameters.
       <xs:element minOccurs="0" name="id-max-length" type="ValueUInt-CT"/>
       <xs:element minOccurs="0" name="id-quotes" type="ValueString-CT"/>
       <xs:element minOccurs="0" name="start-of-week-offset" type="ValueUInt-CT"/>
+      <xs:element minOccurs="0" name="start-of-week-format" type="StartOfWeekFormat-CT"/>
```
Comment (rosswbrown) [inline validation/tdd_latest.xsd:650]:
```suggestion
      <xs:element minOccurs="0" name="start-of-week-format" type="StartOfWeekFormat-CT"/> <!-- added 2021.3 -->
```

File: validation/tdd_latest.xsd [hunk 3]
```diff
@@ -215,6 +215,8 @@ as passed-in parameters.
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="IIF"/>
+      <xs:enumeration value="IIF_"/>
+      <xs:enumeration value="IF"/>
```
Comment (rosswbrown) [inline validation/tdd_latest.xsd:219]:
```suggestion
      <xs:enumeration value="IIF_"/> <!-- added 2021.3 -->
      <xs:enumeration value="IF"/> <!-- added 2021.3 -->
```

**Review by rosswbrown — COMMENTED** [review]
I noticed we added this version comment in some other new addition.  Seems handy, we'll see if we can keep it up for now.

File: validation/tdd_latest.xsd (hunk 1 above)
Comment (rosswbrown) [reply validation/tdd_latest.xsd:561]: This should say 2021.3

**Review by sxlin — APPROVED** [review]
Ship it after add the comments Ross suggested.

**Comment by jsun28** [comment]
> I noticed we added this version comment in some other new addition. Seems handy, we'll see if we can keep it up for now.

Added comments as you suggested, and I tested it with packager, validation is working fine.

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**